### PR TITLE
Fixed default settings for domain access fields.

### PR DESCRIPTION
**Pull request by BasilSnowman**
Fixed default settings for domain access fields.

### CODE REVIEW

File: domain_menu_access.module [hunk 1]
```diff
@@ -41,16 +42,19 @@ function domain_menu_access_form_menu_link_content_form_alter(&$form, FormStateI
       '#open' => TRUE,
       '#weight' => 25,
     );
-
+  
+    $domains = Domain::loadMultiple();
+    $options = array_map(function($domain) {
```
Comment (andypost) [inline domain_menu_access.module]: does it loads disabled domains?

File: domain_menu_access.module [hunk 2]
```diff
@@ -41,21 +43,39 @@ function domain_menu_access_form_menu_link_content_form_alter(&$form, FormStateI
...
     $form[DOMAIN_ACCESS_ALL_FIELD]['#access'] = FALSE;
   }
 }
 
 /**
+ * Implements hook_entity_field_access_alter().
+ */
+function domain_menu_access_entity_field_access_alter(array &$grants, array $context) {
+  if (
+    (
```
Comment (andypost) [inline domain_menu_access.module]: useless line

File: domain_menu_access.module (hunk 1 above)
Comment (BasilSnowman) [reply domain_menu_access.module]: it's excess. removed.

File: domain_menu_access.module [hunk 3]
```diff
@@ -41,17 +46,14 @@ function domain_menu_access_form_menu_link_content_form_alter(&$form, FormStateI
       '#open' => TRUE,
       '#weight' => 25,
     );
-
+  
+    $domains = Domain::loadMultiple();
+    $options = array_map(function($domain) {
+      return $domain->getDomainId();
```
Comment (andypost) [inline domain_menu_access.module]: are you sure that domainId() should be there, I bet it should different + totally useless now

**Pull request closed without merging**